lib/generators/sorcery/templates/initializer.rb

```diff
@@ -281,6 +281,16 @@
     # user.subclasses_inherit_config =
 
     # -- remember_me --
+    # change default remember_me_token attribute.
```

Comment:
Looks like this was accidentally included from your code in the other PR.

Reply:
It is the column name of the `remember_me_token` attribute, not the name of the remember me cookie.